bpf-prog/bpf-tc/include/helper.h

```diff
@@ -0,0 +1,18 @@
+#include <netinet/in.h>
+#include <stdbool.h>
+
+#define L4_HDR_OFF (ETH_HLEN + sizeof(struct ipv6hdr))
+#define BPF_F_PSEUDO_HDR (1ULL << 4)
+
+static __always_inline bool compare_ipv6_addr(const struct in6_addr *addr1, const struct in6_addr *addr2)
+{
+#pragma unroll
```

Comment:
what does `#pragma unroll` mean?

Reply:
@tamilmani1989 Likely a directive to force the compiler to unroll loops. I'm speculating that it's probably necessary (along with `__always_inline`) to convince the BPF verifier that loops terminate.